TrackForce/src/test/java/com/revature/test/restAssured/CurriculumResourceTest.java

```diff
@@ -75,51 +87,61 @@ public void testGetAllCurriculumsUnhappyPath() {
 		assertTrue(response.statusCode() == 401);
 		assertTrue(response.asString().contains("Unauthorized"));
 
-		given().header("Authorization", tokenAdmin).when().get(URL + "badurl/").then().assertThat().statusCode(404);
+		given().header("Authorization", tokenAdmin).when().get(URL + "/badurl").then().assertThat().statusCode(404);
```

Comment:
Going along with the use of EnvManager, the URL convention is slashes at the end of each section of a URL, so should be "badurl/"